src/Admin/Models/OrganizationEditModel.cs

```diff
@@ -59,8 +60,10 @@ public class OrganizationEditModel : OrganizationViewModel
     public string BraintreeMerchantId { get; set; }
 
     [Required]
-    [Display(Name = "Name")]
+    [Display(Name = "Organization Name")]
     public string Name { get; set; }
+    [Display(Name = "Client Owner Email")]
+    public string ClientOwnerEmail { get; set; }
```

Comment:
Is this shown anywhere? I think it can probably be removed from this PR and done as part of EC-435, just to keep things separate.

Reply:
It is not! I've removed it.